src/tooling/docs-builder/Http/DocumentationWebHost.cs

```diff
@@ -152,6 +153,12 @@ await context.Response.WriteAsync(@"
 					FileProvider = new EmbeddedOrPhysicalFileProvider(Context),
 					RequestPath = "/_static"
 				})
+			.UseStaticFiles(
```

Comment:
We should ensure `.svg files are picked up as `ImageFile`'s and rely on `ServeDocumentationFile()` to serve these as static. 

Assuming this was added to serve the `svg` files?